server/graphql/common/expenses.ts

```diff
@@ -1180,7 +1191,16 @@ async function payExpenseWithPayPalAdaptive(remoteUser, expense, host, paymentMe
 }
 
 export async function createTransferWiseTransactionsAndUpdateExpense({ host, expense, data, fees, remoteUser }) {
-  await createTransactions(host, expense, fees, data);
+  if (host.settings?.transferwise?.ignorePaymentProcessorFees) {
+    // TODO: We should not just ignore fees, they should be recorded as a transaction from the host to the collective
+    // See https://github.com/opencollective/opencollective/issues/5113
+    fees.paymentProcessorFeeInHostCurrency = 0;
+  } else if (data?.paymentOption?.fee?.total) {
+    fees.paymentProcessorFeeInHostCurrency.fees = Math.round(data.paymentOption.fee.total * 100);
+  }
+
+  const fxRate = 1; // TODO
```

Comment:
@kewitz Let's talk about this one, I'm not sure where's the best place to retrieve this info (the quote?)

Reply:
Correct me if I'm wrong, but...

We always store the DEBIT transaction in the same currency as the collective, which can differ from the host in some edge cases.
We want to store the CREDIT transaction in the same currency of the expense, which now can be any* currency the user picks and can also differ from the collective.

For DEBIT:
- We have the actual value that was debited from the Host's Wise account.
- Use getFxRate() between host and collective.

For CREDIT:
- Use have the actual FX rate we have in expense.data.quote because it should be the actual rate used between Host and User.